src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java

```diff
@@ -87,15 +86,15 @@ public void setUp() throws IOException {
     @Test
     public void scrapeChannelUploadPlaylist_nonExistentChannelId() throws IOException {
         mockChannelResponse = new ChannelListResponse();
```

Comment:
1) mockChannelResponse is being initialized every time it's referenced, so it doesn't need to be a field.  make it a local variable. (intellij would have warned you that it could be a local variable vs. a field).  in general mutable fields, because mutable state = bad.
2)  i know what you're trying to say.  it not a real response from the YouTube API server, but strictly speaking it's not a mock or a fake.     it's a *real* ChannelListResponse, but it's a test one so maybe have a local var called testChannelResponse, so rename everywhere.